public/app/features/dashboard/components/PanelEditor/OptionsPaneCategory.tsx

```diff
@@ -115,15 +115,15 @@ export const OptionsPaneCategory = React.memo(
         className={boxStyles}
         data-testid="options-category"
         data-plugin-sandbox={sandboxId}
-        aria-label={selectors.components.OptionsGroup.group(id)}
+        data-testid={selectors.components.OptionsGroup.group(id)}
```

Comment:
Duplicate `data-testid` on this div. It's difficult to know which is the least disruptive one to remove...

Lets remove `data-testid="options-category"` from this.

Reply:
did this, can you direct me on how you are checking all the tests locally?

Reply:
about breaking this PR into series of smaller PR's. Will this work for now? or does it require it?